ase/CMakeLists.txt

```diff
@@ -25,35 +25,59 @@
 ## POSSIBILITY OF SUCH DAMAGE.
 
 ## Install target for ASE support files
+project(ase)
 set(ASE_SHARE_DIR share/opae/ase)
+add_subdirectory(api)
 
-install(DIRECTORY rtl     DESTINATION share/opae/ase COMPONENT asertl)
-install(DIRECTORY sw      DESTINATION share/opae/ase COMPONENT asesw)
-install(DIRECTORY scripts DESTINATION share/opae/ase USE_SOURCE_PERMISSIONS COMPONENT asescripts)
+install(DIRECTORY rtl          DESTINATION share/opae/ase COMPONENT asertl)
+install(DIRECTORY sw           DESTINATION share/opae/ase COMPONENT asesw)
+install(DIRECTORY scripts      DESTINATION share/opae/ase USE_SOURCE_PERMISSIONS COMPONENT asescripts)
 
+# Also needed
+configure_file(${PROJECT_SOURCE_DIR}/in/CMakeLists.txt.in
+  ${PROJECT_BINARY_DIR}/CMakeLists.txt)
```

Comment:
@abelardojarab What is the ${PROJECT_BINARY_DIR} value? A build folder under ASE?